package-lock.json

```diff
@@ -2201,17 +2201,6 @@
         "node": ">=6.0.0"
       }
     },
-    "node_modules/@jridgewell/source-map": {
```

Comment:
The changes to this file should be dropped (unless there's a reason for them that I'm not seeing!)